src/mcts/node.h

```diff
@@ -280,9 +297,9 @@ class Node {
 
 // A basic sanity check. This must be adjusted when Node members are adjusted.
 #if defined(__i386__) || (defined(__arm__) && !defined(__aarch64__))
-static_assert(sizeof(Node) == 40, "Unexpected size of Node for 32bit compile");
+static_assert(sizeof(Node) == 48, "Unexpected size of Node for 32bit compile");
```

Comment:
There is a pointer and a 32bit number.  On the 64 bit platform the 32 bit number disappears into the existing wasted padding, but on 32 I am pretty sure it will not.

Reply:
Sorry, I overlooked the 32-bit number.